learn/advanced/faceted_search.md

```diff
@@ -0,0 +1,159 @@
+      "BEERBONG .COM":1,
+      "Dr. Pet":1,
+      …
+      "senover":1
+    },
+    "rating":{
+      "0":7,
+      …
+      "4.8":1,
+      "5":3
+    }
+  }
+  …
+}
+```
+
+`facetDistribution` contains an object for every given facet. For each of these facets, there is another object containing all the different values and the count of matching documents. Note that zero values will not be returned: if there are no `pie server` products by a certain brand, it is not displayed.
+
+::: note
+By default, `facets` returns a maximum of 100 facet values for each faceted field. You can change this value using the `maxValuesPerFacet` property of the [`faceting` index settings](/reference/api/settings.md#faceting).
+:::
+
+### Facet stats
+
+When using the `facets` parameter, any matching documents with facets containing numeric values are displayed in a `facetStats` object. `facetStats` contains the numeric minimum (`min`) and maximum (`max`) values per facet for all documents matching the search query.
+
+If none of the matching documents have a numeric value for a facet, that facet is not included in the `facetStats` object.
+
+Meilisearch ignores string values like `"21"` when computing `facetStats`.
+
+The following response shows the `facetStats` when searching for `pie server`:
+
+```json
+{
+  …
+  "facetStats":{
+    "rating":{
+      "min":0.0,
+      "max":5.0
+    }
+  }
+  …
+}  
+```
+
+Since `rating` was the only numeric facet in our example, it is returned in the `facetStats` object.
+
+## Facet types
+
+### Single select facets
+
+Single-select facets allow the selection of a single value per facet. Using our `ecommerce` index, this means if you have a facet for `brand` with the values `BrandA`, `BrandB`, and `BrandC`. Single select facets will allow you to view products from one brand at a time. If you want to view `BrandA` and `BrandB` products, you will have to try both searches individually. If you want to further narrow down results using the `reviews` facet, you would need to do so separately for both brands.
+
+### Multi-select facets
+
+Multi-select facets allow users to choose multiple options within a facet so they don’t have to perform more than one search to find the products they’re looking for.
+
+With single select facets, if a user selects `BrandA` for the `brands` facet, they will see it offers 5 products, nothing more. But with multi-select facets, we want to show users how many products `BrandB` and `BrandC` offer even when they're not selected. This way the user knows if other brands offer similar products and can view products from multiple brands at the same time.
+
+The image below shows multi-select facets in action. Even though the user selected the `Lithonia Lighting` and `Sunlite` brands when searching for `LED`, they can see other brands and how many LED products they have.
+
+![Meilisearch demo for multi-select facets searching for 'LED'](/faceted-search/facets-multi-select.png)
```

Comment:
This section should mirror the single-select facets section as much as possible:

1. define multi-select search
2. explain use-case
3. illustrate use-case (and refer back to single-select if needed)